### PR TITLE
Horovod documentation update

### PR DESCRIPTION
**Pull request by Setol21**
Hi!

Just added the basic knowledge about implementing horovod into a python script with tensorflow v2 keras.

Once again, really really thank you for your guide through.

### CODE REVIEW

File: docs/source/horovod.rst [hunk 1]
```diff
@@ -45,4 +38,96 @@ If you choose the `tensorflow2` backend ::
...
+1. ::
+
+    CUDA_VISIBLE_DEVICES="0,1"
+
+This part allows you to pass certain gpus to the ``horovodrun`` command, in this case we are using the gpu 
+``0`` and ``1``.
+
+2. ::
+
+    horovodrun -np 2 -H localhost:2
```
Comment (pescap) [inline docs/source/horovod.rst:55]: I would put steps 2 and 3 together, as they are the same command:

```suggestion
    horovodrun -np 2 -H localhost:2 python tensorflow2_synthetic_benchmark.py
```

Comment (pescap) [reply docs/source/horovod.rst:55]: Just reformulate a little, to put those two items together.

File: docs/source/horovod.rst [hunk 2]
```diff
@@ -45,4 +38,96 @@ If you choose the `tensorflow2` backend ::
...
+3. ::
+
+    python tensorflow2_synthetic_benchmark.py
+
+And finally you use python to run the wanted file.
+
+
+.. Later on in this part we will add the parallel to DeepXDE.
+
+Implementing horovod
```
Comment (pescap) [inline docs/source/horovod.rst:69]:
```suggestion
Use horovod
```

**Comment by pescap** [comment]
Very good job, thank you Daniel!

**Comment by Setol21** [comment]
The requested changes has been applied.

**Pull request merged**